cmd/gencerts_test.go

```diff
@@ -0,0 +1,122 @@
+package cmd_test
+
+import (
+	"crypto/x509"
+	"encoding/pem"
+	"io/ioutil"
+	"os"
+	"path"
+	"testing"
+
+	"github.com/OpenBazaar/openbazaar-go/cmd"
+	"github.com/OpenBazaar/openbazaar-go/schema"
+)
+
+func buildCertDirectory() (*string, func(), error) {
+	ctx := schema.SchemaContext{
+		DataPath:        schema.GenerateTempPath(),
+		TestModeEnabled: true,
+	}
+	certSchema := schema.MustNewCustomSchemaManager(ctx)
+	if err := certSchema.BuildSchemaDirectories(); err != nil {
+		return nil, nil, err
+	}
+	return &ctx.DataPath, certSchema.DestroySchemaDirectories, nil
+}
+
+func TestGenCertsGenericDefaults(t *testing.T) {
+	var dataPath, Destroy, schema_err = buildCertDirectory()
```

Comment:
Typically, we use capitalized names to indicate which variables are exported publicly from the package. Variables cannot be exported out of the function scope so we try to keep all function-scoped variable lowercase to not confuse readers. Idomatic go also prefers camelcase over snakecase.

Please correct:
`var datapath, destroy, schemaErr = buildCertDirectory()`